src/store.js

```diff
@@ -393,27 +395,31 @@ function makeLocalContext (store, namespace, path) {
   return local
 }
 
+let makeLocalGettersCache = {}
```

Comment:
Can we place the cache into store instance so that we can avoid global mutable variable?